### PR TITLE
ENH: Add support for itk.PointSet

### LINKED ISSUE #595: Support for viewing `itk.PointSet` in `main` branch
## Summary

I am unable to directly view an `itk.PointSet` in `itkwidgets>=1.0a21`. This functionality is available in `itkwidgets==0.32.5`. Uncertain if this is as intended, or if it could be added to the itkwidgets roadmap.

## Steps to Reproduce

```
input_points = itk.PointSet[itk.F,3].New()
# ... insert several points into the set
itkwidgets.view(point_sets=[input_points])
# or
itkwidgets.view(point_sets=input_points)
```

## Expected Behavior

An output window with points is displayed for `itkwidgets==0.32.5`.

## Observed Behavior

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[14], line 1
----> 1 view(point_sets=input_points)

File ~\Anaconda3\envs\venv-itk\lib\site-packages\itkwidgets\viewer.py:436, in view(data, **kwargs)
    322 def view(data=None, **kwargs):
    323     """View the image and/or point sets.
    324 
    325     Creates and returns an ImJoy plugin ipywidget to visualize an image, and/or
   (...)
    434         properties on the object to change the visualization.
    435     """
--> 436     viewer = Viewer(data=data, **kwargs)
    438     return viewer

File ~\Anaconda3\envs\venv-itk\lib\site-packages\itkwidgets\viewer.py:136, in Viewer.__init__(self, ui_collapsed, rotate, ui, **add_data_kwargs)
    132 def __init__(
    133     self, ui_collapsed=True, rotate=False, ui="pydata-sphinx", **add_data_kwargs
    134 ):
    135     input_data = self.input_data(add_data_kwargs)
--> 136     data = self.init_data(input_data)
    137     """Create a viewer."""
    138     self.viewer_rpc = ViewerRPC(
    139         ui_collapsed=ui_collapsed, rotate=rotate, ui=ui, data=data
    140     )

File ~\Anaconda3\envs\venv-itk\lib\site-packages\itkwidgets\viewer.py:179, in Viewer.init_data(self, input_data)
    177         result = _get_viewer_point_sets(data)
    178     if result is None:
--> 179         raise RuntimeError(f"Could not process the viewer {input_type}")
    180     _init_data[key] = result
    181 return _init_data

RuntimeError: Could not process the viewer point_sets
```

## Workaround

A NumPy array can be constructed from the `itk.PointSet` and then viewed as in the example https://github.com/InsightSoftwareConsortium/itkwidgets/blob/main/examples/NumPyArrayPointSet.ipynb

### PR DESCRIPTION
**Pull request by bnmajor**
Fixes #595

### CODE REVIEW

File: itkwidgets/integrations/__init__.py [hunk 1]
```diff
@@ -173,6 +177,9 @@ def _detect_render_type(data, input_type) -> RenderType:
         import itk
         if isinstance(data, itk.Image):
             return RenderType.IMAGE
+        elif isinstance(data, itk.PointSet):
+            print(f'isinstance(data, itk.PointSet)', flush=True)
```
Comment (thewtex) [inline itkwidgets/integrations/__init__.py:181]: remove debug code

**Review by thewtex — CHANGES_REQUESTED** [review]
@bnmajor thank you! 

Could we also please add a test notebook?

Perhaps an example from the master branch or an adaptation of https://examples.itk.org/src/core/transform/deformavolumewithathinplatespline/documentation?highlight=plate ?

File: itkwidgets/integrations/__init__.py (hunk 1 above)
Comment (bnmajor) [reply itkwidgets/integrations/__init__.py]: Fixed, thanks!

**Review by thewtex — APPROVED** [review]
@bnmajor thanks!!

**Pull request merged**